src/main/scala/gitbucket/core/controller/ControllerBase.scala

```diff
@@ -244,4 +244,12 @@ trait AccountManagementControllerBase extends ControllerBase {
         .map    { _ => "Mail address is already registered." }
   }
 
+  protected def reservedNames(): Constraint = new Constraint(){
+    override def validate(name: String, value: String, messages: Messages): Option[String] = if(value == "git"){
```

Comment:
Some other names such as `admin`, `upload` and `api` are reserved actually:
https://github.com/gitbucket/gitbucket/blob/master/src/main/scala/ScalatraBootstrap.scala#L30-L32

Could you also add them?